sdk/core/core-rest-pipeline/README.md

```diff
@@ -32,15 +32,19 @@ A `PipelineResponse` describes the HTTP response (body, headers, and status code
 A `SendRequest` method is a method that given a `PipelineRequest` can asynchronously return a `PipelineResponse`.
 
 ```ts snippet:send_request
-export type SendRequest = (request: PipelineRequest) => Promise<PipelineResponse>;
+import { PipelineRequest, PipelineResponse } from "@azure/core-rest-pipeline";
```

Comment:
unfortunately "export" isn't allowed in methods.  Also imports are pulled in.  This is core so not really customer facing though. 

If we prefer I can revert the change and add // @ts-ignore